label_studio_converter/imports/yolo.py

```diff
@@ -131,38 +158,60 @@ def add_parser(subparsers):
     yolo = subparsers.add_parser('yolo')
 
     yolo.add_argument(
-        '-i', '--input', dest='input', required=True,
+        '-i',
+        '--input',
+        dest='input',
+        required=True,
         help='directory with YOLO where images, labels, notes.json are located',
-        action=ExpandFullPath
+        action=ExpandFullPath,
     )
     yolo.add_argument(
-        '-o', '--output', dest='output',
+        '-o',
+        '--output',
+        dest='output',
         help='output file with Label Studio JSON tasks',
         default='output.json',
-        action=ExpandFullPath
+        action=ExpandFullPath,
     )
     yolo.add_argument(
-        '--to-name', dest='to_name',
+        '--to-name',
+        dest='to_name',
         help='object name from Label Studio labeling config',
         default='image',
     )
     yolo.add_argument(
-        '--from-name', dest='from_name',
+        '--from-name',
+        dest='from_name',
         help='control tag name from Label Studio labeling config',
         default='label',
     )
     yolo.add_argument(
-        '--out-type', dest='out_type',
+        '--out-type',
+        dest='out_type',
         help='annotation type - "annotations" or "predictions"',
         default='annotations',
     )
     yolo.add_argument(
-        '--image-root-url', dest='image_root_url',
+        '--image-root-url',
+        dest='image_root_url',
         help='root URL path where images will be hosted, e.g.: http://example.com/images',
         default='/data/local-files/?d=',
     )
     yolo.add_argument(
-        '--image-ext', dest='image_ext',
+        '--image-ext',
+        dest='image_ext',
         help='image extension to search: .jpeg or .jpg, .png',
         default='.jpg',
     )
+    yolo.add_argument(
+        '--image-dims',
+        dest='image_dims',
+        help=(
+            "optional tuple of integers specifying the image width and height of *all* "
+            "images in the dataset. Defaults to opening the image to determine it's width "
+            "and height, which is slower. This should only be used in the special "
+            "case where you dataset has uniform image dimesions. e.g. `--image-dims 600 800` "
```

Comment:
Not sure it's working - `--image-dims 600 800` - because 600 800 has a space in between..

Reply:
Good catch, sorry for the delay! Tests incoming

Reply:
@Axel-Jacobsen Do you have any news about tests? Will you be able to add them?

Reply:
Ah sorry! It has been hectic at work, I'll work on these this week pinky promise

Reply:
yes, awesome!